CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Features
 - Add focus styles for `Menu.Item` component @Bugaa92 ([#286](https://github.com/stardust-ui/react/pull/286))
 
+### Documentation
+- Add `Prototypes` section and `Chat Pane` prototype @Bugaa92 ([#235](https://github.com/stardust-ui/react/pull/235))
+
 <!--------------------------------[ v0.8.0 ]------------------------------- -->
```

Comment:
Please add somewhere here that this is under development mode flag at this point.

Reply:
done